docs/vnext/openapi.yaml

```diff
@@ -347,7 +264,7 @@ components:
         type: string

```

Comment:
Should we support an optional version header for extensions to return in responses that we could include in our logging?

Reply:
Makes sense. Added the header.